views/eApostilles/applicationSubmissionSuccessful.ejs

```diff
@@ -0,0 +1,5 @@
+<h1 class="heading-xlarge">Application successfully submitted</h1>
+
+<section class="column-two-thirds">
+
+</section>
```

Comment:
is this required?

Reply:
Good question, so this is just to have something to show when the payment is successful. It will be fleshed out in the future ticket but this is just a placeholder. So not required for this feature but helpful for the overall flow.

Reply:
worth commenting out or deleting entirely. take a look at the "YAGNI" principle 👀

Reply:
I'm learning a lot from you on this PR, "YAGNI" principle 🤦  The things that people come up with.

Reply:
but aside from that, you make a good point :)

Reply:
even though it's a handful of KB, it could be sitting collecting dust until the feature is prioritised, and you wont know when.  If another developer picks it up, they might think it was put there for a reason, and they'd create another <section/> just in case that empty <section/> breaks something/is required for something.